lib/findpath.mjs

```diff
@@ -0,0 +1,44 @@
+import { dirname } from 'node:path';
+import { env, platform } from 'node:process';
+import { join, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+/**
+ * Get the platform dependant path of the NW.js binary.
+ * 
+ * @param {'nwjs' | 'chromedriver'} exe Path to NW.js or Chromedriver executable.
+ * @return {string}
+ */
+export default function findpath(exe = 'nwjs') {
```

Comment:
You should create two new functions `findNw` and `findChromedriver`. Each that just focus on their own logic. Then `findpath` only needs to worry about which to call based on what `exe` is equal to. Also, it is best to avoid abbreviating variables so that code is consistent and predictable (`exe` => `executable`).